proto/xudrpc.proto

```diff
@@ -442,12 +449,33 @@ message ListPeersResponse {
 message ListTradesRequest {
   // The maximum number of trades to return
   int32 limit = 1 [json_name = "limit"];
-} 
+}
 
 message ListTradesResponse {
   repeated Trade trades = 1 [json_name = "trades"];
 }
 
+message ListSwapsRequest {
+  int32 limit = 1 [json_name = "all"];
+  RequestedSwapState requested_swap_state = 2 [json_name = "requested_swap_state"];
+  enum RequestedSwapState {
+    BOTH = 0;
```

Comment:
I'm thinking that we will have "pending" deals as well. No need to deal with those right now, but calling this option `ALL` instead of `BOTH` might make more sense and cover us if we have more than 2 statuses to choose from later on.